client/my-sites/checkout/checkout/checkout-container.jsx

```diff
@@ -12,24 +14,47 @@ import Checkout from '../checkout';
 import CartData from 'components/data/cart';
 import CheckoutData from 'components/data/checkout';
 import SecondaryCart from '../cart/secondary-cart';
+import SignupSiteCreatedNotice from 'my-sites/checkout/checkout/signup-site-created-notice';
 
 /**
  * Style dependencies
  */
 import './checkout-container.scss';
 
+/**
+ * Returns whether given site has been created in the last `n` minutes
+ *
+ * @param  {String}  createdAt               The site creation date stamp
+ * @param  {Number}  creationWindowInMinutes A site is considered 'new' if it's been created in this time window (in minutes)
+ * @return {Boolean}                         If the site is 'new'. Default `false`
+ */
+function isSelectedSiteNew( createdAt, creationWindowInMinutes = 5 ) {
```

Comment:
I agree.

`isNewSite` [exists](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/selectors/is-new-site.js). 

I thought about using that, but it seemed like an overkill at the time since I have the selectedSite object to work with immediately, and don't have to reach into the state and so on.